### PR TITLE
IAM integrations - GitHub

### PR DESCRIPTION
**Pull request by ohaim1008**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/31189

## Description
Github integration for the IAM project.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Demisto
- [ ] 5.0.0
- [ ] 5.5.0
- [x] 6.0.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

File: Packs/GitHub/Integrations/GitHub_IT_Admin/GitHub_IT_Admin.py [hunk 1]
```diff
@@ -0,0 +1,426 @@
...
+            json_data=data,
+            ok_codes=ok_codes
+        )
+
+        return res
+
+    def get_user(self, input_type, user_term):
+
+        user_term = "\"" + user_term + "\""
+        uri = f'scim/v2/organizations/{encode_string_results(self.org)}/' \
```
Comment (anara123) [inline Packs/GitHub/Integrations/GitHub_IT_Admin/GitHub_IT_Admin.py:47]: why are you using encode_string_results everywhere?

File: Packs/GitHub/Integrations/GitHub_IT_Admin/GitHub_IT_Admin.py [hunk 2]
```diff
@@ -0,0 +1,426 @@
...
+    res = client.http_request(method='GET', url_suffix=uri)
+    res_text = res.text
+    if 'documentation_url' in res_text:
+        errortext = "URL or Organization Name " + str(client.org) + " Not Found."
+        raise Exception(str(res.status_code) + " " + str(errortext))
+    else:
+        uri = ""
+        res = client.http_request(method='GET', url_suffix=uri)
+        if res.status_code == 200:
+            return 'ok', None, None
```
Comment (anara123) [inline Packs/GitHub/Integrations/GitHub_IT_Admin/GitHub_IT_Admin.py:143]: seems that we return results in test_module but those results are not passed to `return_results` and we should return simple text from test_module.

**Review by anara123 — CHANGES_REQUESTED** [review]

**Review by anara123 — CHANGES_REQUESTED** [review]
missing detailed description file

File: Packs/GitHub/Integrations/GitHub_IT_Admin/GitHub_IT_Admin.yml [hunk 3]
```diff
@@ -0,0 +1,299 @@
...
+      type: String
+  - arguments:
+    - default: false
+      description: A User Profile indicator.
+      isArray: false
+      name: user-profile
+      required: true
+      secret: false
+    deprecated: false
+    description: Disable an active user.
```
Comment (anara123) [inline Packs/GitHub/Integrations/GitHub_IT_Admin/GitHub_IT_Admin.yml:247]:
```suggestion
    description: Deletes an active user.
```

**Review by anara123 — APPROVED** [review]
Feel free to merge when you are ready

File: Packs/GitHub/ReleaseNotes/1_1_2.md [hunk 4]
```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+##### New: GitHub IAM
+- Integrate with GitHub's services to perform Identity Lifecycle Management operations.
+- Upgraded the Docker image to demisto/python3:3.8.6.13358.
```
Comment (kirbles19) [inline Packs/GitHub/ReleaseNotes/1_1_2.md:5]: Do we need to show that we updated the Docker image if it's a brand new integration? If we can (and should) remove this, remove the "-" in front of the sentence beginning with "Integrate with..."

**Review by kirbles19 — APPROVED** [review]

**Comment by kirbles19** [comment]
@ohaim1008 Done.

File: Packs/GitHub/ReleaseNotes/1_1_2.md (hunk 4 above)
Comment (ohaim1008) [reply Packs/GitHub/ReleaseNotes/1_1_2.md:5]: Sounds good.

**Pull request merged**